crates/types/src/services/p2p.rs

```diff
@@ -72,6 +78,108 @@ pub struct GossipData<T> {
 /// Transactions gossiped by peers for inclusion into a block
 pub type TransactionGossipData = GossipData<Transaction>;
 
+/// Transactions that have been confirmed by block producer
+pub type ConfirmationsGossipData = GossipData<PreConfirmationMessage>;
+
+/// A value and an associated signature
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct Sealed<Entity> {
+    /// The actual value
+    pub entity: Entity,
+    /// Seal
+    pub signature: Signature,
```

Comment:
Nit: I'd suggest renaming the signature field to `seal` or renaming the type to `Signed` so the naming is more consistent. There's also a "todo" to do the same renaming in the sibling struct in the `blockchain/consensus.rs` module.
```suggestion
    /// Seal
    pub seal: Signature,
```

Reply:
I prefer `Signature`. `Seal` is ambiguous IMO.

Reply:
Should the type then be called `Signed` instead of `Sealed` perhaps?